### ISSUE TITLE
broken WCAG explainer page

### ISSUE THREAD

**Issue by danielweck** [daisy/kb#65]
http://kb.daisy.org/publishing/docs/wcag/index.html (or http://kb.daisy.org/publishing/docs/wcag/ )

linked from: http://kb.daisy.org/publishing/docs/topics.html

**Comment by danielweck** [comment]
```
Uncaught TypeError: Cannot read properties of null (reading 'insertAdjacentElement')
    at KB.generatePageTitle (init.js:396:39)
    at KB.writeTemplate (init.js:169:6)
    at window.onload (init.js:1407:5)
```

**Comment by mattgarrish** [comment]
Thanks for the report @danielweck! The page was missing a piece of metadata to indicate it is an index (triggers slightly different formatting). I've fixed it in https://github.com/daisy/kb/commit/36d44ffd5a1eaaa8de26b34a244e0414c2146c08
